liveblog/liveblog
# [LBSD-2772] Fix author on syndicated blog not showing up properly

### Purpose
To fix the issue that shows the wrong author on syndicated posts after these have been edited in the producer's side. 

### What has changed
- Adjustments made to the mechanism that sends the post to the consumer in order to attach the right creator's data
- Code refactor of syndication webhook code to make it easier to understand and maintain 
- Refactor of themes' variables name to give them a better meaning (from `item` to `post` in `template-post.html`)
- Adjusted themes to show the right author taking information from the syndicated_creator attribute instead of the first item of the post

### Steps to test

In order to test this PR you will need to set up Content Syndication in your local instance. You will first need to register a "Consumer" and then a "Producer" that would point out into your own local instance. Please read the wiki document about [Content Syndication](https://sofab.atlassian.net/wiki/x/hIFdPwI) to get a better understanding of it.

- Checkout the branch
- Set up Content Syndication
- Create a Producer blog (enable syndication in blog settings)
- Create a Consumer blog and attach the Producer blog (you can enable automatic publish)
- Create a post in Producer, see it ingested from the consumer 
- Then edit the same post with a different user. Observe that the author should remain the same both in Editor's UI and in the embed side

For the review, I would recommend you to hide the whitespace so there is less noise in the PR: 
| <img width="450" alt="image" src="https://github.com/liveblog/liveblog/assets/124766/185ce2b1-0952-45e9-9693-b1aca1b29ca7"> |
| - |

Please let me know if you have any issues while trying to test it. 

Resolves: [LBSD-2772]

Thanks for checking!

[LBSD-2772]: https://sofab.atlassian.net/browse/LBSD-2772?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
